assets/src/edit-story/components/moveable/cropStyle.js

```diff
@@ -19,127 +19,166 @@
 	}
+	.crop-moveable .moveable-control.moveable-w {
+	  margin-top: 0 !important;
+	}
+
+
 
 	.crop-moveable .moveable-control.moveable-e,
 	.crop-moveable .moveable-control.moveable-w {
-		width: 4px !important;
-		height: 16px !important;
-		margin-left: -1px !important;
-		margin-top: -8px !important;
+		width: ${SIDE_NARROW}px !important;
+		height: ${SIDE_WIDE}px !important;
+		margin-top: -12px !important;
 	}
 
 	.crop-moveable .moveable-control.moveable-nw,
 	.crop-moveable .moveable-control.moveable-ne,
 	.crop-moveable .moveable-control.moveable-sw,
 	.crop-moveable .moveable-control.moveable-se {
-		width: 16px !important;
-		height: 16px !important;
-		background: #fff !important;
+		width: ${SIDE_WIDE}px !important;
+		height: ${SIDE_WIDE}px !important;
+		background: ${({ theme }) =>
+      theme.designSystemTheme.colors.blue[20]} !important;
 		position: absolute !important;
+		border-radius: 2px;
 	}
 
 	.crop-moveable .moveable-control.moveable-nw::before,
 	.crop-moveable .moveable-control.moveable-ne::before,
 	.crop-moveable .moveable-control.moveable-sw::before,
 	.crop-moveable .moveable-control.moveable-se::before {
 		content: "" !important;
-		width: 16px !important;
-		height: 16px !important;
+		width: ${SIDE_WIDE}px !important;
+		height: ${SIDE_WIDE}px !important;
 		display: block !important;
 		position: absolute !important;
 		inset: 1px !important;
-		background: #000 !important;
+		background: ${({ theme }) =>
+      theme.designSystemTheme.colors.bg.primary} !important;
 	}
 
 	.crop-moveable .moveable-control.moveable-nw,
 	.crop-moveable .moveable-control.moveable-ne {
-		margin-top: -2px !important;
+		margin-top: -8px !important;
 	}
 
 	.crop-moveable .moveable-control.moveable-sw,
 	.crop-moveable .moveable-control.moveable-se {
-		margin-top: -14px !important;
+		margin-top: -16px !important;
 	}
 
 	.crop-moveable .moveable-control.moveable-nw,
 	.crop-moveable .moveable-control.moveable-sw {
-		margin-left: -2px !important;
+		margin-left: -8px !important;
 	}
 
 	.crop-moveable .moveable-control.moveable-ne,
 	.crop-moveable .moveable-control.moveable-se {
-		margin-left: -14px !important;
-	}
+		margin-left: -16px !important;
+	}
+
+	.crop-moveable .moveable-line.moveable-direction[data-line-index="0"] {
+    margin-top: -4px;
+  }
+  .crop-moveable .moveable-line.moveable-direction[data-line-index="3"] {
+    margin-top: 4px;
+  }
+  .crop-moveable .moveable-line.moveable-direction[data-line-index="1"] {
+    margin-left: 4px;
+  }
+  .crop-moveable .moveable-line.moveable-direction[data-line-index="2"] {
+    margin-left: -4px;
+  }
 
 	.crop-moveable .moveable-control.moveable-nw {
 		transform-origin: 2px 2px !important;
-		clip-path: polygon(0 0, 16px 0, 16px 4px, 4px 4px, 4px 16px, 0 16px) !important;
+		clip-path: polygon(0 0, ${SIDE_WIDE}px 0, ${SIDE_WIDE}px ${SIDE_NARROW}px, ${SIDE_NARROW}px ${SIDE_NARROW}px, ${SIDE_NARROW}px ${SIDE_WIDE}px, 0 ${SIDE_WIDE}px) !important;
```

Comment:
This doesn't look correct. It's close, but not really there - two corners are missing rounding and the inner corners (on the shape itself, not the border) are not rounded at all.

Let's change it to 3 shapes - the primary element and both `::before` and `::after`, styled like this:
![Screen Shot 2021-02-08 at 19 09 24](https://user-images.githubusercontent.com/637548/107297435-59fb6c00-6a41-11eb-80b7-d8f228a94ad8.png)

I've tried styling the corners using this idea in c0bd7a6f8, and I like it (well, better at least).

| Design | Clip-path | `::before` + `::after` |
|-|-|-|
| ![Screen Shot 2021-02-08 at 19 59 54](https://user-images.githubusercontent.com/637548/107301006-97172c80-6a48-11eb-8e27-18f470406f64.png) | ![Screen Shot 2021-02-08 at 19 59 41](https://user-images.githubusercontent.com/637548/107301008-97172c80-6a48-11eb-8faa-ff9418bed3b5.png) | ![Screen Shot 2021-02-08 at 20 01 41](https://user-images.githubusercontent.com/637548/107301005-967e9600-6a48-11eb-9b04-55bc42c5c9a0.png) |

Yes yes, I know. It's tiny details that nobody will notice. But I did! 😁